tests/test_localhost_integ.c

```diff
@@ -408,12 +408,6 @@ static int s_localhost_integ_h2_upload_stress(struct aws_allocator *allocator, v
     s_tester.alloc = allocator;
 
     size_t length = 2500000000UL;
-#ifdef AWS_OS_LINUX
```

Comment:
remove this, since it seems to be the flow control window issue.
1. we sent out too much
2. the initial window from the local server is small (default to the magic 65535). https://httpwg.org/specs/rfc7540.html#iana-settings
3. The local server code also update the window as data received.